### DISCUSSION TITLE
Table LoadingContent stays at the first column

### DISCUSSION THREAD

**Discussion by yul761** [heroui-inc/heroui#3349] [Help]
Implement a table. the loadingContent is somehow stays within the first column (first `<td>`) instead of render in the row.

Like this:
![Screenshot 2024-06-27 at 4 11 49 PM](https://github.com/nextui-org/nextui/assets/15523223/655e78fd-4b0b-4a10-8402-5cdef3a1a4a9)

Causing the loading spinner unable to stay at the center of the table
Like this
![image](https://github.com/nextui-org/nextui/assets/15523223/24508b1a-45a1-49c3-93cc-d130d682cbf5)

Here is the sample code piece:
```
<Table aria-label="Example static collection table">
        <TableHeader>
          <TableColumn>Mint Time</TableColumn>
          <TableColumn>File Name</TableColumn>
          <TableColumn>Action</TableColumn>
        </TableHeader>
        <TableBody
          isLoading={loading}
          items={items}
          loadingContent={<p>loading...</p>}
          emptyContent={"No rows to display."}
        >
          {(item) => (
            <TableRow>
              <TableCell>{item.mintTime.toString()}</TableCell>
              <TableCell>{item.fileName}</TableCell>
              <TableCell>
                  <IconButton style={{ color: "white" }}>
                    <PreviewIcon />
                  </IconButton>
              </TableCell>
            </TableRow>
          )}
        </TableBody>
      </Table>
```

could anyone help review it to see what goes wrong? The loading content could show and disappear at the correct time, just the position is off. I am looking to render the loadingContent directly under `<tr>` so i could style its position.

Thanks

**Comment by wingkwong** [comment]
Can you share which version are you using?

**Reply by yul761** [reply]
I am using 2.0.35. I am importing it from the `/table` though, not import it from `@nextui-org/react`. Don't know if that will cause this issue
` "@nextui-org/table": "^2.0.35",`
